ach/src/main/java/com/adyen/checkout/ach/ACHDirectDebitConfiguration.kt

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by onurk on 24/1/2023.
+ */
+
+package com.adyen.checkout.ach
+
+import android.content.Context
+import com.adyen.checkout.action.ActionHandlingPaymentMethodConfigurationBuilder
+import com.adyen.checkout.action.GenericActionConfiguration
+import com.adyen.checkout.components.base.ButtonConfiguration
+import com.adyen.checkout.components.base.ButtonConfigurationBuilder
+import com.adyen.checkout.components.base.Configuration
+import com.adyen.checkout.components.model.payments.Amount
+import com.adyen.checkout.core.api.Environment
+import kotlinx.parcelize.Parcelize
+import java.util.Locale
+
+@Parcelize
+@Suppress("LongParameterList")
+class ACHDirectDebitConfiguration private constructor(
+    override val shopperLocale: Locale,
+    override val environment: Environment,
+    override val clientKey: String,
+    override val isAnalyticsEnabled: Boolean?,
+    override val amount: Amount,
+    override val isSubmitButtonVisible: Boolean?,
+    internal val genericActionConfiguration: GenericActionConfiguration,
+    val addressConfiguration: AddressConfiguration,
+) : Configuration, ButtonConfiguration {
+
+    class Builder :
+        ActionHandlingPaymentMethodConfigurationBuilder<ACHDirectDebitConfiguration, Builder>,
+        ButtonConfigurationBuilder {
+
+        private var isSubmitButtonVisible: Boolean? = null
+        private var addressConfiguration: AddressConfiguration =
+            AddressConfiguration.FullAddress(supportedCountryCodes = listOf("US", "PR"))
```

Comment:
Is this kept here by mistake or it's supposed to be the default value?

Reply:
They are default values

Reply:
Then they should only be handled in the component params mapper, and `addressConfiguration` here would be nullable (check how we do it in the card configuration for example)